lib/pf/provisioner/mobileconfig.pm

```diff
@@ -274,6 +271,25 @@ sub ca_cert_cn {
     }
 }
 
+=head2 get_other_radiusd_certificates
+
+Get Other Radius server CA Certificates
+
+=cut
+
+sub get_other_radiusd_certificates {
```

Comment:
Why not modify all callers of  get_other_radiusd_certificates to do what's done in the return statement? Nothing else is done in that function so we could trash it.

Reply:
I used it there to make a difference between provisionner and radius.

So for me:
* get_cn_and_cert_radiusd_certificates belongs to radius and can be used for other purposes than the provisionner.
* get_other_radiusd_certificates is just a representation of get_cn_and_cert_radiusd_certificates in the provisionner context with the only objective to fill the XML with info.

Still, there are commented lines (created to represent get_cn_and_cert_radiusd_certificates in provisioning context) that can be removed since @jrouzierinverse find that it was only a [ ] missing.

Does it make sens?

Reply:
It's not that it doesn't make sense, it's just that the function basically "does nothing". I'm OK if you want to keep it for the rationale above, just at least remove the commented content.

Reply:
Ok got it. Just removed commented lines